printer_mods/Slidr/PanzerBalls2.4/readme.md

```diff
@@ -0,0 +1,23 @@
+What are Panzerballs?
+- Panzerballs are another type of feet using commonly available squash balls.
+
+Why do they exist?
+- i hate sourcing of rubber compressor feet
+- i felt that other squash ball mods do not satisfy me visually (sry)
+
+Why do i need Panzerballs?
+- You probably don´t need them unless you got problems with sourcing the normal rubber compressor feet 
+or you want something more squishy with maybe a better noise reduction.
+
+How do i install Panzerballs?
+- Panzerballs are a drop in replacement for the base plate
+- You need:
+  - 4x squash balls (super slow, double yellow dot, diameter 39.5-40.5 mm)
+    o german source: https://www.decathlon.de/p/squashballe-sb-990-2er-pack-zwei-gelbe-punkte/_/R-p-312192?currentPage=1&filter=all&mc=8572487&orderId=de2200321911
+  - 4x printed Panzerballs:
+    o Use '01-Normal\[a]_PanzerBall_x4.stl' if you have not modded your z-Drive
+    o If you use edwardyeeks V2.4_z_drive_tensioner_mod (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/edwardyeeks/V2.4_z_drive_motor_tensioner_mod)
```

Comment:
Please make this a relative link

    [V2.4_z_drive_tensioner_mod](../../edwardyeeks/V2.4_z_drive_motor_tensioner_mod)